jit-integration-tests/src/test/java/org/dcsa/jit/itests/Main.java

```diff
@@ -0,0 +1,47 @@
+package org.dcsa.jit.itests;
```

Comment:
In integration tests everything should actually go under main so it's included in the jar that it builds

Reply:
I do not understand the remark.  I hope it was that you wanted the HealthCheck check like in OVS.  Otherwise, I have no clue what it meant.

Reply:
src/test/{java,resources} -> src/main/{java,resources}. Stuff in src/test is not included in the jar

Reply:
ah, that "main". Fixed.